### DISCUSSION TITLE
Dynamic colors for different types of containers

### DISCUSSION THREAD

**Discussion by modernpacifist** [i3/i3#5799] [Q&A] [answered]
Is there a way to explicitly assign colors for tiling and floating containers?
The result that I'm trying to achieve is for floating containers to be different colors from tiling containers.

**Comment by orestisfl** [comment, marked as answer]
That's not possible, colors are the same for all containers and not set per-container.

**Reply by modernpacifist** [reply]
That is sad. Might this be considered to be added in the future as a feature?
